pennylane/transforms/sign_expand/sign_expand.py

```diff
@@ -194,13 +195,10 @@ def construct_sgn_circuit(  # pylint: disable=too-many-arguments
     return tapes
 
 
-batch_transform_with_diff_false = partial(batch_transform, differentiable=False)
```

Comment:
what was the value-add of `differentiable=False`, and have we changed any functionality here by dropping it?

Reply:
From what I can gather, it sets gradient_fn to none before the execute. I think it can be misleading, because non-differentiability does not mean that you have no gradient function or that all the results are 0.